mdc

```diff
@@ -66,27 +85,34 @@ for mode in ${MODES}; do
     COMPOSE_PROFILES="${COMPOSE_PROFILES},MODE_${mode}"
 
     # if there is a mode specific env file then include it
-    efiles="${MODES_DIR}/${mode}/env ${MODES_DIR}/${mode}/.env"
+    efiles="${path}/env ${path}/.env"
     for efile in ${efiles}; do
         if [ -e "${efile}" ]; then
-            echo "### mdc begin mode ${mode}" >> ${ENV_FILE}-mdc-tmp
+            echo "### mdc begin mode ${mode} (${path})" >> ${ENV_FILE}-mdc-tmp
             vecho "cat ${efile} >> ${ENV_FILE}-mdc-tmp"
             cat ${efile} >> ${ENV_FILE}-mdc-tmp
             echo "### mdc end mode ${mode}" >> ${ENV_FILE}-mdc-tmp
```

Comment:
Maybe add `($path)` here too for completeness.

Reply:
Done.  Also switched to `(${efile})` for both.  It seems very slightly better to me to make it truly unambiguous and reduce the knowledge of this code on how things were found.